### ISSUE TITLE
Q: difference from tabbedex

### ISSUE THREAD

**Issue by laur89** [gryf/tabbedalt#17]
Sorry for this daft of a question, but could you summarize main difference from https://github.com/mina86/urxvt-tabbedex, if you're aware of it?

**Comment by gryf** [comment]
The main key feature of tabbedalt is the **named tabs**. This was the sole reason I've started to changing original tabbed extension from urxvt package. The other differences are the matter of taste, I guess. Over time I started to add things, which I saw on other implementations, and which I'd like to have.

**Comment by laur89** [comment]
Not quite sure I get the feature from reading the readme. Is it basically automatic tab renaming logic?

**Comment by gryf** [comment]
No. By default, creating a tab (shell), you get the `shell` titled tab (as in screenshots on the readme file). You can create number of custom shortcuts pointing to other things like different shells or run specified commands,  i.e.:

```
URxvt.keysym.Control-Shift-T: tabbedalt.new_tab:top:htop
URxvt.keysym.Control-Shift-D: tabbedalt.new_tab:some-host:ssh myhost
```

After reloading X resources you'll be able to run `htop` with ctrl+shift+t shortcut in new tab named „htop“, and make a ssh connection to the myhost host with shortcut ctrl+shift+d, which create tab titled „some-host“, so that I don't have to guess which tab out of 12 existing contain `mc`. Of course you can use rename feature to change names on fly, but you'll have the habit of renaming newly created tabs right away to make that work.

**Comment by laur89** [comment]
Ah gotcha. Sounds like an interesting idea. Guess the trade-off is between remembering to rename a tab right after opening it, and remembering new set of shortcuts.
Thanks!
